### PR TITLE
SPCTR-3253: Fix - Post layout typography text transform default/normal is not working as expected. 

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Post layout - Typography  transform Normal/Default option not working.
https://brainstormforce.atlassian.net/browse/SPCTR-3253

Issue: `text-transform: normal`  style is not  CSS standard property so I have changed normal -> none.
https://d.pr/i/9wPEu7

### Screenshots

https://d.pr/v/OqdiNy

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by AbhijitNage123 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -221,6 +221,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 * Fix: Global - Resolved an issue wherein a block error would occur when attempting to select typography font families in the presence of multiple iFrames.
 * Fix: Global - Resolved an issue wherein adding Containers, Forms, Lottie blocks and the Popup Builder caused errors when the Gutenberg plugin was active.
 * Fix: Lottie - Resolved an issue in which the width of the block would be adopted from the parent Container.
+* Fix: Post Grid - Resolved an issue in which the meta text transform has no variation between capitalize and normal.
```
Comment (bsf-zanev) [inline README.md:224]: Isn't this fix Global?

If so:
```suggestion
* Fix: Global - Resolved an issue wherein normal text transformation would not work as intended.
```

**Review by bsf-zanev — COMMENTED** [review]
Is this a Global fix?

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: Done.

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

**Review by adi3890 — APPROVED** [review]

**Pull request merged**